.nvmrc

```diff
@@ -0,0 +1 @@
+0.10.13
```

Comment:
It does. Travis performs the following: `nvm install $(< .nvmrc)` when installing node on the build run. Locally, you just run `nvm use`.